lattice-control/.github/workflows/lattice-control-interface.yml

```diff
@@ -0,0 +1,55 @@
+name: LATTICE_CONTROL_INTERFACE
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+    - "rust/**"
+    tags:
+    - 'lattice-control-interface-v*'
+  pull_request:
+    branches: [ main ]
+    paths:
+    - "rust/**"
+
+env:
+  CARGO_TERM_COLOR: always
+  working-directory: ./rust
```

Comment:
This should be `./lattice-control/rust` here

Reply:
thanks great catches on these.